### ISSUE TITLE
Rounds should timeout if people disappear before answering

### ISSUE THREAD

**Issue by Longsight** [grifferz/pah-irc#48]
[14:36:02] <AgainstHumanity> → A romantic, candlelit dinner would be incomplete without _____.
[14:36:02] <AgainstHumanity> Now message me your answers please!
[14:38:53] <AgainstHumanity> Current Black Card:
[14:38:53] <AgainstHumanity> → A romantic, candlelit dinner would be incomplete without _____.
[14:40:30] <AgainstHumanity> 3 people just played! We're currently waiting on plays from 2 people:
[14:44:42] <AgainstHumanity> 1 person just played! We're currently waiting on plays from 2 people: grifferz and dominic.

...and still waiting.

**Comment by grifferz** [comment]
There is already a config item called `turnclock` which says how long a turn can be idle before someone gets punished (kicked out).

The intention for this bot (see https://github.com/grifferz/pah-irc/blob/master/README.md) is for it to be slow-paced. Tons of near-real-time Cards Against Humanity IRC bots already exist and I didn't want this to be another one. So, I envision `turnclock` to be on the order of 1 or 2 **days**.

Right now, for testing purposes, there is an instance of the bot in #debug on Blitzed where the `turnclock` is set to 1 hour. It seems to keep 4 to 6 people playing all the time during UK waking hours.

Real-time play is obviously popular and I'm not totally opposed to it, but until I am convinced that slow play cannot work I will prioritise those feature requests lower.

So in summary, this setting already exists, you just happen to be playing in a game which has it set longer than you would like.
